packages/office-addin-dev-settings/src/cli.ts

```diff
@@ -78,13 +78,13 @@ commander
 commander
     .command("webview <manifest-path> [web-view-type]")
     .description("Allows specifying a preference for the type of web view to use when debugging. Windows only")
-    .action(commands.webView)
+    .action(commands.webViewOverride)
```

Comment:
Maybe I gave poor guidance earlier but I would be OK with the function webView() being for the "webview" command. It was the exported functions that I wanted to have a better name. Hope I didn't cause confusion.

Reply:
Funny enough, I actually now prefer webViewOverride to my previous name choice! Thank you though!